### ISSUE TITLE
EmptyRule linter doesn't find empty rules if they use polyfill-unscoped-rule

### ISSUE THREAD

**Issue by leiboldm** [brigade/scss-lint#844]
Example empty rule that isn't caught:

```
polyfill-unscoped-rule {
  content: '.some-class-name';

}
```

**Comment by sds** [comment]
Hey @leiboldm, thanks for the report.

While I don't know anything significant about the Polymer project (which a quick search tells me this `polyfill-unscoped-rule` comes from), it appears to me to be a third-party (read: non-standardized) extension to CSS (if that's not the case, please point me to some documentation that indicates otherwise).

I would rather not introduce custom logic for this third-party framework. Feel free to create a [custom SCSS-Lint linter](https://github.com/brigade/scss-lint#custom-linters) in your repository to handle this particular case. Thanks!
